### PR TITLE
Added static data to inital request payload

### PR DESCRIPTION
**Pull request by nathan-weinberg**

### CODE REVIEW

File: server/mbta_api.py [hunk 1]
```diff
@@ -231,11 +237,13 @@ async def initial_request_data(route_ids, test_mode=False):
     )
     sightings = get_recent_sightings_for_lines()
     git_tag = get_git_tag()
+    static_data = get_static_data()
```
Comment (idreyn) [inline server/mbta_api.py:240]: Could we move this call outside of the `initial_request_data` function? That way we don't have to `open()` this file every time we want to serve the page — we should only need to do it once.

**Review by idreyn — APPROVED** [review]
Don't want to block you but see my comment!

**Comment by nathan-weinberg** [comment]
Going to modify this PR to use https://docs.python.org/3/library/functools.html

**Comment by austinjpaul** [comment]
I know I'm late to the party here, but what about letting the server serve the static json file as is, and then just unpacking it client-side?

**Comment by nathan-weinberg** [comment]
@idreyn ^^ WDYT?

**Comment by idreyn** [comment]
I think we should load what we can in the initial response through `initial_data`, unless it's very large (which the stats tab data may end up being?) Fewer fetches to keep track of and more amenable to server-side rendering.

**Review by austinjpaul — APPROVED** [review]
fine for now.
as discussed previously, if the file grow much larger, we should consider other approaches.
(load it into memory once instead of with each request? let nginx handle it?)

**Pull request merged**